discord/appinfo.py

```diff
@@ -292,6 +320,18 @@ def icon(self) -> Optional[Asset]:
             return None
         return Asset._from_icon(self._state, self.id, self._icon, path='app')
 
+    @property
+    def cover_image(self) -> Optional[Asset]:
+        """Optional[:class:`.Asset`]: Retrieves the cover image of the application's default rich presence.
+
+        This is only available if the application is a game sold on Discord.
```

Comment:
These no longer exist.

Reply:
When using `/applications/@me` I still get a hash for `cover_image`: